src/migrations/hotbar-store/index.ts

```diff
@@ -26,11 +26,11 @@ import { joinMigrations } from "../helpers";
 import version500alpha0 from "./5.0.0-alpha.0";
 import version500alpha2 from "./5.0.0-alpha.2";
 import version500beta5 from "./5.0.0-beta.5";
-import version500beta11 from "./5.0.0-beta.11";
+import version500beta10 from "./5.0.0-beta.10";
```

Comment:
Should we change version number here as well? 
https://github.com/lensapp/lens/blob/master/src/migrations/hotbar-store/5.0.0-beta.11.ts#L38